event_database/functions_server/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM docker.io/golang:1.17.0@sha256:06e92e576fc7a7067a268d47727f3083c0a564331bfc
 
 WORKDIR /app
 
-ADD . .
+COPY . .
```

Comment:
Not a huge fan of ". ." usage when copying, it has the potential to leak secrets when publishing images.  However, since we are not doing this, it's probably ok for the time being.

Reply:
We should aim to be explicit as possible in the relative context we are working in.

Reply:
We can use `.dockerignore` files that could contain things like `.env*`